auto_nag/scripts/patch_closed_bug.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag import utils
+from auto_nag.bzcleaner import BzCleaner
+
+COMMENT_BODY = "A patch has been landed on this ticket, but it has already been closed. Filing a separate ticket will ensure better tracking. If this was not by mistake and further action is needed, please alert the appropriate party."
```

Comment:
```suggestion
COMMENT_BODY = "A patch has been attached on this bug, which was already closed. Filing a separate bug will ensure better tracking. If this was not by mistake and further action is needed, please alert the appropriate party."
```

`landed` is for when a patch is added to the repository.

Reply:
Thank you for the clarification.